appendix-basic65.tex

```diff
@@ -7396,7 +7392,7 @@ \subsection{RPLAY}
 \begin{description}[leftmargin=2cm,style=nextline]
 \item [Token:] \$FE \$0F
 \item [Format:] {\bf RPLAY(voice)}
-\item [Usage:] Returns a value of 1 or 0, to indicate if a melody is playing on this voice channel or not.
+\item [Usage:] Returns a value of 0 or 1, to indicate whether a melody is playing on the given voice channel or not.
```

Comment:
Ah, I think the intention behind describing it as 1 or 0 before was to match it up in a respective manner to the tail-end of the sentence. I.e., if the melody is playing on the given voice channel (1), or not (0).

Though you're welcome to re-phrase it in a different way, just hoping to preserve that relationship in some way.